project/Build.scala

```diff
@@ -1,14 +1,14 @@
 import sbt._
 import Keys._
-import PlayProject._
+import play.Project._
 
 object ApplicationBuild extends Build {
 
     val appName         = "collins"
-    val appVersion      = "1.2.4-SNAPSHOT"
+    val appVersion      = "1.2.5"
```

Comment:
this should already be 1.3.0, and I would suggest we bump this up to 1.4.0 with the play upgrades